src/story/story.service.spec.ts

```
@@ -70,6 +71,7 @@ describe('StoryService - Library & Generation', () => {
           provide: 'CACHE_MANAGER',
           useValue: { del: jest.fn(), get: jest.fn(), set: jest.fn() },
         },
+        { provide: GuestSessionService, useValue: { getGuestSession: jest.fn().mockResolvedValue({ id: 'guest-1' }) } },
```

ESLint

[error] Replace `·provide:·GuestSessionService,·useValue:·{·getGuestSession:·jest.fn().mockResolvedValue({·id:·'guest-1'·})·}` with `⏎··········provide:·GuestSessionService,⏎··········useValue:·{⏎············getGuestSession:·jest.fn().mockResolvedValue({·id:·'guest-1'·}),⏎··········},⏎·······`

(prettier/prettier)